---
title:  "Life after the Synthetic Cell"
author: "Geoffrey Thomson"
date:   2015-10-12
tags: ["Synthetic-biology"]
---

> Note: This is a old piece from about 2012 but I still like it.

In mid-2010 headlines were created around the world when The J. Craig Venter Institute announced what they term the “assembly” of a self replicating cell with a chemically synthesized genome. This was achieved by the *in vitro* reproduction of the 1.08 megabase genome of *Mycoplasma mycoides*, a ruminant parasite and the subsequent transplantation into a *M. capricolum* cell with its genome removed. In addition the authors added some “watermarks” in the form of their names, an email address and some pertinent quotes translated into amino acids to prove that they had done it [^1].The production of a self replicating cell with a chemically synthesized genome is a flagship paper in what has become the exciting field of synthetic biology, here I discuss where science is focusing next.

The principle aims of synthetic biology are threefold. Firstly, to determine the basis of life, or more specifically, the core components required for a (proto-)organism to be classified as alive. Hence the Richard Feynman quote "What I cannot build, I cannot understand" in the “wartermarks” of the synthetic cell is very appropriate. The current theoretical minimum is approximated at 100-151 genes [^2] and *Mycoplasma* was selected as a model organism to study this question because it has the smallest genome of any organism that can be cultured in the lab (organisms with smaller genomes do exist, but are often endosymbionts and/or cannot be cultured). Previous work has found *M. genitalium* has a minimal genome of 382 essential genes [^3], albeit in a lab situation. No doubt as the environment changes different (and larger) sets of genes become essential.

In light of this work, it was very exciting when in July of this year researchers at Stanford University, in collaboration with The J. Craig Venter Institute, announced their whole-cell computational model of *M. genitalium* incorporating a diverse range of experimental datasets. This “first draft” of the model encompasses the full life cycle of the cell and takes the full 525 genes in the genome as well as all of the organism’s molecular components and known interactions in 28 integrated sub-models [^4].

Where the production of a cell with a chemically synthesised genome required the digitisation of the genome, the digitisation of the entire cell is an impressive technical achievement, however it also has important practical applications. These centre on the models phenotypic predictions given the genotype. For example, when analysing the behaviour of DNA-binding proteins the model predicts that 50% of the bacterial chromosome interacts with at least one protein in the first six minutes, 90% in the first 20 minutes and 90% of genes being expressed in the first 143 minutes [^4]. This type of analysis at a holistic cell wide level and scale allows the identification of emergent phenomena. In *M. genitalium* it appears that dNTP metabolism may impose restrictions on the duration of the cell cycle not previously identified [^4].

Given the 525 genes, the whole-cell computational model of *M. genitalium* seems ideally placed to assess the question of a minimal genome. Through simulated gene disruptions the model can delve into why a gene product is needed in the cell system giving new insights. It suggests that there are 284 essential  genes which agrees with the experimental results with 79% accuracy [^4]. This shortfall is actually informative as incorrect predictions allow the targeted identification of pathways and interactions not currently known or understood. Thus indicating the gaps in scientific knowledge and where experimental work should focus next.

This leads on to the second aim of synthetic biology, the illumination of gene function by the expression in novel contexts. This is already a mainstay of contemporary biotechnology however it is limited to a few genes at a time and it is anticipated that the developing techniques of synthetic biology will accelerate this process.

The logical next step in the synthetic biology process is naturally larger and more complex systems, which is relevant to the third aim of synthetic biology, practical biological engineering applications. The next cells to be produced with (at least partially) chemically synthesised genomes will likely be first *E. coli* and then eukaryotes, most probably single-celled algae with the intention of developing algae biofuels [^5]. In fact, Synthetic Genomics Inc., Craig Venter’s private company for synthetic biology purchased 81 acres of southern California in May of this year with the explicit intention of developing engineered strains of algae, funded by ExxonMobil and BP among others [^6].

Of course, synthetic biology has applications beyond biofuels, although this is definitely one of the most promising areas. Hydrocarbon recovery, water purification, agricultural improvement, hydrogen production, vaccine development, pharmaceutical production, whole cell biosensors and targeted therapies are all ideas which have been floated [^7].

There is much work being carried out to achieve these goals most of which are trying to apply classical engineering concepts to these complex systems, principally “modularity, component testing, standards, interfaces, libraries of parts and computer-aided design” [^2]. Naturally this centres around the production of Boolean-like logic structures to construct circuits from modules like a regular electrical circuit or software program. These include switches and oscillators and many researchers have the aim of creating an inventory of wetware components that can be substituted in or out as needed [^2] <sup>&</sup> [^7]. The idea of developing specialized programming languages for biological systems to abstractly design systems is a novel and innovative idea [^2].

However any of this work must take into consideration the range of factors inherent in biological systems such as noise, life-cycle, interactions, feedback mechanisms, component matching, mutations, variations in environmental conditions and cellular context to name a few [^2]. An interesting decision that must be made is whether, in order to function in the real world, these synthetic organisms should have their function be rigidly dictated by their design or should a level of dynamism be built in? Given that biological systems are dynamic the answers are not straightforward.

Breaking a living cell down into its constituent parts is not the approach to synthetic biology and other groups of researchers are taking some different approaches. Broadly speaking these can be classified as “bottom up” approaches as opposed to “top down” and can be thought of as grouping into three general categories.

The first approach comes from the research programs studying the origin of life on Earth. Simply put, labs such as that of Jack Szostak (2009 Nobel Laureate) aim to reproduce the processes that resulted in the shift from chemical evolution to biological evolution on the early Earth by investigating replicating vesicles, RNA and protein selection [^8]. The idea of defining life may not help in making clear the origin of life [^9] but understanding the origin of life may help understanding the minimal requirements for life.

A similar but subtlety different approach is the construction of life from scratch, with the explicit aim of developing a global understanding of cellular life by the artificial construction of a cell from its basic structural elements. These include both genetic and non-genetic processes. In a sense, this is the complementary approach to the creation of a cell with a chemically synthesized genome in that it is concerned with the creation of cellular “hardware” not “software.” After all while the physical structure of a cell is regulated by its genetic material, the cell with a chemically synthesized genome still initially required existing cellular machinery. One prominent aspect of this approach’s practitioners is concerned with the construction of artificial organelles (e.g. ribosomes).

An excellent example of this approach is the artificial cell project which aims to construct a minimal cell and takes John Von Neumann’s theoretical work on replicating automata to study self-reproducing systems. To date, the project has achieved transient simple gene expression in synthetic vesicles [^10]. It would be interesting to see whether Stephen Wolfram’s work on cellular automata has anything to contribute to this line of investigation.

The third category of “bottom up” synthetic biology is much more undefined. It asks why should cells be the smallest basic unit of life and explores a much broader understanding of what life can be [^11]. It is the study of self-organizing processes in natural and human made systems. The majority of this field is theoretical and consists of mathematical speculation and/or computer simulations. This area has a large overlap with astrobiology (or xenobiology to use a better term) [^12].

Synthetic biology is obviously an area where ethical and security risks need to be at the forefront of the discussion [^1] <sup>&</sup> [^7], however each case will need to be considered on a case by case basis. It is difficult to make general statements. It is inevitable that there will be claims of “playing god,” an interesting claim to unpack. However most observers consider that “few, if any, new ethical issues are raised” [^13]. Nevertheless, the prominence in the public eye provides a good opportunity to engage and communicate with the public which can only be a good thing.

As an aside, it is observe that the phrase “playing god” is much more commonly used by non-religious critics of biotechnology than religious groups [^14] and that the God of “playing God” may not actually be the God of the Bible, but rather “deified nature” [^15]. Synthetic biology therefore has profound philosophical implications to be explored as well with regards to our relationship to the natural world.

There are many other topics that the field of synthetic biology has yet to tackle. Chief among these are the issues surrounding multi-cellularity and the phenomena of emergence. Synthetic biology of higher levels of biological organization have not been discussed, such as synthetic ecosystems, however the applicable fields (e.g. invasion ecology) are significantly advanced and too board to be summarized here. Lastly, there are many small novel projects which only time will tell what impact they will have, one of particular note is intracellular data storage which uses a “rewritable recombinase addressable data module” to use the genetic code as a sort of binary data storage [^16]. Nevertheless the ultimate goal of synthetic biology is the *de novo* scripting of genetic code. When this is achieved the result will be very exciting to see.

[^1]: Gibson et al. (2010). <i>Science</i>. doi:<a href="https://www.sciencemag.org/content/329/5987/52">10.1126/science.1190719</a>
[^2]: Purnick and Weiss. (2009). <i>Nature Reviews Molecular Cell Biology</i>. doi:<a href="http://www.nature.com/nrm/journal/v10/n6/full/nrm2698.html">10.1038/nrm2698</a>
[^3]: Glass et al. (2006). <i>PNAS</i>. doi:<a href="http://www.pnas.org/content/103/2/425">10.1073/pnas.0510013103</a>
[^4]: Karr et al. (2012). <i>Cell</i>. doi:<a href="http://www.cell.com/abstract/S0092-8674%2812%2900776-3">10.1016/j.cell.2012.05.044</a>
[^5]: Actually with hindsight see Annaluru et al. (2014). <i>Science</i>. doi:<a href="http://www.sciencemag.org/content/344/6179/55">10.1126/science.1249252</a>
[^6]: <a href="http://www.syntheticgenomics.com/240512.html">Synthetic Genomics Inc. Press Release 24/5/2012</a>
[^7]: Khalil and Collins. (2010). <i>Nature Reviews Genetics</i>. doi:<a href="http://www.nature.com/nrg/journal/v11/n5/full/nrg2775.html">10.1038/nrg2775</a>
[^8]: Schrum, Zhu and Szostak. (2010). <i>Cold Spring Harbor Perspectives in Biology</i>. doi:<a href="http://cshperspectives.cshlp.org/content/2/9/a002212">10.1101/cshperspect.a002212</a>
[^9]: Szostak. (2012). <i>Journal of Biomolecular Structure and Dynamics</i>. doi:<a href="http://www.tandfonline.com/doi/abs/10.1080/073911012010524998">10.1080/073911012010524998</a>
[^10]: Noireaux, Maeda and Libchaber. (2010). <i>PNAS</i>. doi:<a href="http://www.pnas.org/content/108/9/3473">    10.1073/pnas.1017075108</a>
[^11]: Tirard, Morange and Lazcano. (2010). <i>Astrobiology</i>. doi:<a href="http://online.liebertpub.com/doi/abs/10.1089/ast.2010.0535">10.1089/ast.2010.0535</a>
[^12]: Cohen, and Stewart. (2001). <i>Nature</i>. doi:<a href="http://www.nature.com/nature/journal/v409/n6823/full/4091119a0.html">10.1038/35059248</a>
[^13]: Cho and Relman. (2010). <i>Science</i>. doi:<a href="https://www.sciencemag.org/content/329/5987/38">10.1126/science.1193749</a>
[^14]: Link. (2013). <i>Science and Engineering Ethics</i>. doi:<a href="http://link.springer.com/article/10.1007%2Fs11948-012-9353-z">10.1007/s11948-012-9353-z</a>
[^15]: van den Belt. (2009). <i>NanoEthics</i>. doi:<a href="http://link.springer.com/article/10.1007%2Fs11569-009-0079-6">10.1007/s11569-009-0079-6</a>
[^16]: Bonnet, Subsoontorn and Endy. (2012). <i>PNAS</i>. doi:<a href="http://www.pnas.org/content/109/23/8884">10.1073/pnas.1202344109 </a>

